app/src/main/java/timur/gilfanov/messenger/domain/usecase/user/ChangeUiLanguageError.kt

```diff
@@ -3,9 +3,26 @@ package timur.gilfanov.messenger.domain.usecase.user
 import timur.gilfanov.messenger.domain.usecase.user.repository.ChangeLanguageRepositoryError
 
 /**
- * Errors for UI language change operations.
+ * Errors that can occur during UI language change operations.
  *
- * Type alias to [ChangeLanguageRepositoryError] as use case layer has no unique errors.
- * If use case-specific errors are needed, convert this to a sealed interface.
+ * Represents failures at the use case layer, combining identity retrieval errors
+ * with repository-level language change errors.
  */
-typealias ChangeUiLanguageError = ChangeLanguageRepositoryError
+sealed interface ChangeUiLanguageError {
+    /**
+     * Failed to retrieve current user identity.
+     *
+     * This error occurs when [IdentityRepository] cannot provide the identity needed
+     * to perform the language change operation.
+     */
+    data object IdentityError : ChangeUiLanguageError
```

Comment:
Object need to be more specific about what happened with identity to handle error properly.
```suggestion
    data object NoIdentityError : ChangeUiLanguageError
```